src/components/form/templates/types.ts

```diff
@@ -31,7 +31,7 @@ export interface ObjectFieldTemplateProps extends TemplateProps {
     description: string;
     properties: ObjectFieldProperty[];
     idSchema: any;
-    formData: object;
+    formData: any;
```

Comment:
I believe it can be undefined or null. I forget exactly but yes it can be.

Reply:
If that's the case we could have it like this instead
```
formData?: object;
```
`any` would mean that it could be numbers and strings as well, and I don't think thats true :thinking:

Reply:
Ill have to double check but i believe it would be true that they could be anything. Since Fields can override at any level of the json schema, it could override a string fields and then formData would be a string i believe

Reply:
Just double checked. It can be a string. Likely can be an integer too.